packages/nextjs/src/common/withServerActionInstrumentation.ts

```diff
@@ -113,7 +113,10 @@ async function withServerActionInstrumentationImplementation<A extends (...args:
                 formDataObject[key] = '[non-string value]';
               }
             });
-            span?.setData('server_action_form_data', formDataObject);
+            // Since formDataObject is not a primitive, we cannot store it on the span as attributes
+            // so instead we put it as extra on the scope
+            const scope = getCurrentScope();
+            scope.setExtra('server_action_form_data', formDataObject);
```

Comment:
@lforst do we need this as a nested object here? attributes cannot be nested, so going forward we need to either flatten this, if the shape is known (which I guess it is not?), or store it as context on the scope (?), or as JSON string...